authors/urls.py

```diff
@@ -18,12 +18,10 @@
 from django.urls import path
 from rest_framework_swagger.views import get_swagger_view
 
-schema_view = get_swagger_view(title='Authors Haven API')
-
-
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('api/', include('authors.apps.authentication.urls', namespace='authentication')),
-    path('', schema_view),
+    path(
```

Comment:
also for this one.

Reply:
I'll revert back to the previous commit and redo the task because there is a lot of files that was edited